Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -968,7 +991,8 @@ extension HTTPClient {
         /// - returns: The value of ``futureResult`` when it completes.
         /// - throws: The error value of ``futureResult`` if it errors.
         @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
-        public func get() async throws -> Response {
+        @preconcurrency
+        public func get() async throws -> Response where Response: Sendable {
```

Comment:
These `where` clauses are superfluous, you've required `Response` to be `Sendable`.